datacite-transform/src/main/java/no/unit/nva/transformer/Transformer.java

```diff
@@ -24,14 +24,14 @@
 
     /**
      * Transforms a DynamoDB record to a Datacite Record.
-     * @param dynamoRecordDto A DynamoDB record.
+     * @param dataCiteMetadataDto A DynamoDB record.
```

Comment:
DynamoDB probably should be replaced with DataCiteMetadataDto to hide storage impl.

Reply:
So ensure the javadoc is correct.